### PR TITLE
Add classes to convert a String to a Date

### PR DESCRIPTION
**Pull request by JorgeRCV**

### CODE REVIEW

**Comment by JRojas999** [comment]
You should add the validations of the date string format. In other hand good job.

**Review by JRojas999 — APPROVED** [review]

**Review by JRojas999 — APPROVED** [review]

**Review by JRojas999 — APPROVED** [review]

File: src/main/java/core/date/DateManager.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import org.openqa.selenium.InvalidArgumentException;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class DateManager {
+    private String dateToConvert;
+    public Calendar calendar;
```
Comment (Raymundogs) [inline src/main/java/core/date/DateManager.java:22]: maybe calendar and the formatter should be private.

File: src/main/java/core/date/DateManager.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            calendar.add(dateReader.getTimeUnit(date), -1 * dateReader.getQuantity(date));
+            return calendar.getTime();
+        }
+        if (dateToConvert.contains("from now")) {
+            DateReader dateReader = new DateReader();
+            calendar.add(dateReader.getTimeUnit(date), dateReader.getQuantity(date));
+            return calendar.getTime();
+        }
+        if (dateToConvert.contains("/")) {
+            Date date1 = null;
```
Comment (Raymundogs) [inline src/main/java/core/date/DateManager.java:54]: date1 is not a representative name.

**Comment by JorgeRCV** [comment]
The changes are inluded on the next PR

**Pull request closed without merging**